xarray/tests/test_backends.py

```diff
@@ -1955,6 +1955,38 @@ def test_dump_encodings_h5py(self):
             assert actual.x.encoding['compression_opts'] is None
 
 
+# Requires h5py>2.9.0
```

Comment:
Can you add a `pytest.mark.skipif` based on the version number? (The test on Travis-CI is failing on Python 3.5 because it has an old version of h5py installed)

Reply:
i think i did this correctly (added some lines to `tests/__init__.py`)